python/pdstools/adm/ADMDatamart.py

```diff
@@ -1719,18 +1725,16 @@ def generate_model_reports(
             raise ValueError(
                 "model_list argument is None, not a list, or contains non-string elements for generate_model_reports. Please provide a list of model_id strings to generate reports."
             )
-
-        logger = logging.getLogger(__name__)
         working_dir = Path(working_dir) if working_dir else Path.cwd()
         working_dir.mkdir(parents=True, exist_ok=True)
-
-        # Create a unique temporary directory name
         temp_dir_name = (
             tempfile.mkdtemp(prefix=f"tmp_{name}_", dir=working_dir)
             if name
             else tempfile.mkdtemp(prefix="tmp_", dir=working_dir)
         )
         temp_dir_path = Path(temp_dir_name)
+        timestamp = datetime.datetime.now().strftime("%Y%m%d_%H%M%S")
```

Comment:
I would suggest to use the ISO standard (https://www.iso.org/iso-8601-date-and-time-format.html). Main change is to remove the `_` and add milliseconds.